packages/maskbook/src/plugins/ITO/UI/CreateForm.tsx

```diff
@@ -301,45 +308,36 @@ export function CreateForm(props: CreateFormProps) {
     )
 
     const StartTime = (
-        <LocalizationProvider dateAdapter={AdapterDateFns}>
-            <MobileDateTimePicker
-                showTodayButton
-                ampm={false}
-                label={t('plugin_ito_begin_time', { zone: GMT >= 0 ? `(UTC +${GMT})` : `(UTC ${GMT})` })}
-                onChange={(date: Date | null) => handleStartTime(date!)}
-                renderInput={(props) => <TextField {...props} style={{ width: '100%' }} />}
-                value={startTime}
-                DialogProps={props.dateDialogProps}
-            />
-        </LocalizationProvider>
+        <TextField
+            label={t('plugin_ito_begin_time', { zone: GMT >= 0 ? `(UTC +${GMT})` : `(UTC ${GMT})` })}
+            type="datetime-local"
+            value={formatDateTime(startTime, "yyyy-MM-dd'T'HH:mm")}
+            onChange={(e) => handleStartTime(e.currentTarget.value)}
+            InputLabelProps={{ shrink: true }}
+            inputProps={{ className: classes.datetime }}
+        />
     )
 
     const EndTime = (
-        <LocalizationProvider dateAdapter={AdapterDateFns}>
-            <MobileDateTimePicker
-                showTodayButton
-                ampm={false}
-                label={t('plugin_ito_end_time', { zone: GMT >= 0 ? `(UTC +${GMT})` : `(UTC ${GMT})` })}
-                onChange={(date: Date | null) => handleEndTime(date!)}
-                renderInput={(props) => <TextField {...props} style={{ width: '100%' }} />}
-                value={endTime}
-                DialogProps={props.dateDialogProps}
-            />
-        </LocalizationProvider>
+        <TextField
```

Comment:
I remember that I've created a component named `<DateTimePanel />`. Could you please also update that one?